tests/testReplaceExisting.c

```diff
@@ -73,6 +73,6 @@ int main(int argc, char **argv)
 	}
 
 	json_object_put(my_object);
-
+getch();
```

Comment:
Indent is missing.

Also, why? The commit should have a brief explanation of what was wrong and why this fixes it.